api/shared/nodenetworkconfigurationpolicy_types.go

```diff
@@ -12,18 +14,20 @@ type NodeNetworkConfigurationPolicySpec struct {
 	// The desired configuration of the policy
 	DesiredState State `json:"desiredState,omitempty"`
 
-	// When set to true, changes are applied to all nodes in parallel
+	// MaxUnavailable specifies percentage or number
+	// of machines that can be updating at a time. Default is 1.
```

Comment:
I can set default to 50%, but in CI we only have 3 nodes, so we'll need to force parallel lane to 75% to have 2 nodes working in parallel

Reply:
Done:
- normal lane: NMSTATE_MAXUNAVAILABLE=1
- parallel lane: NMSTATE_MAXUNAVAILABLE=2 (for the reason above, 50% would only be 1 node)

Reply:
Ack, let's do a follow up PR with KUBEVIRT_NUM_NODES=5 (1 master + 4 workers) since most of the test NNCP are applied towards workers and keep just one lane.

Since we will have just one lane even with 5 nodes we will consume less resources (2 lanes * 3 nodes = 6 nodes)